src/pages/DetailsPage.js

```diff
@@ -14,11 +15,24 @@ DetailsPage.propTypes = {
   }),
 }
 
-export default function DetailsPage({ onNavigate, gallery }) {
-  const { image, name, style, address, opening, description } = gallery
+export default function DetailsPage({ onNavigate, gallery, handleBookmark }) {
+  const {
+    id,
+    image,
+    name,
+    style,
+    address,
+    opening,
+    description,
+    isBookmarked,
+  } = gallery
   return (
     <Wrapper>
       <img src={image} alt="" />
```

Comment:
Maybe it would be good to use Cumulative Layout Shift at your images to improve the User Experience.